SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/MechanicalOperations.h

```diff
@@ -136,6 +136,7 @@ class SOFA_SIMULATION_CORE_API MechanicalOperations
     void addSubMBK_ToMatrix(const sofa::core::behavior::MultiMatrixAccessor* matrix, const helper::vector<unsigned> & subMatrixIndex, SReal mFact, SReal bFact, SReal kFact);
 
     void multiVector2BaseVector(core::ConstMultiVecId src, defaulttype::BaseVector *dest, const sofa::core::behavior::MultiMatrixAccessor* matrix);
+    void baseVector2MultiVector(core::MultiVecId dest, defaulttype::BaseVector *src, const sofa::core::behavior::MultiMatrixAccessor* matrix);
```

Comment:
`void baseVector2MultiVector(defaulttype::BaseVector *src, core::MultiVecId dest, const sofa::core::behavior::MultiMatrixAccessor* matrix);`
would be more consistent with multiVector2BaseVector and with the idea of "XXX to YYY", don't you think ?

Reply:
Done